ddtrace/profiling/exporter/http.py

```diff
@@ -109,6 +109,11 @@ def _get_tags(service):
             "runtime_version": PYTHON_VERSION,
             "profiler_version": ddtrace.__version__.encode("utf-8"),
         }
+
+        version = os.environ.get("DD_VERSION")
```

Comment:
you use `os.getenv` below, just pointing out the inconsistency 😆

Reply:
well they're equivalent so… ;)